### ISSUE TITLE
[DRT] Microsuite Failures AppModel & Editing

### ISSUE THREAD

**Issue by grubioe** [dotnet/wpf#1053]
Fix following Microsuite failures:

Feature | Test Case | Reproduced OS | Log | Notes:
-- | -- | -- | -- | --
AppModel | /Name="SplashScreen_Png_CS"   /SubArea="SplashScreen" | All test OSes | SplashScreen_Png_CS | SplashScreen msbuild action is not   supported in Net Core 3
AppModel | /Name="Win7_JumpList_JumpItemsRemovedByUser"   /SubArea="Win7" | Failed OSes:   RS1 ServerStandard,   RS1 x86,   RS3 x64,   RS4 x86,   RS5 x64,   RS6 x86 | Win7_JumpList_JumpItemsRemovedByUser   On .NET Framework, it’s active bug 126798 |  
Editing | /Name="XamlPackageBVTTest"   /SubArea="Clipboard"   /Name="ParagraphEditingWithMouse2"   /SubArea="RichEditing"    /Name="TextRangeSaveLoadTest"   /SubArea="TextOM" | All test OSes | [3Cases]XamlPackageBVTTest&ParagraphEditingWithMouse2&TextRangeSaveLoadTest |

**Comment by rladuca** [comment]
Updating estimate based on the number of failures here.  Generally, if we are considering 2 days per failure this is a better picture of the time this will take.

**Comment by rladuca** [comment]
Re-enabling the editing tests as they are now passing.

See internal PR: https://dnceng.visualstudio.com/internal/_git/dotnet-wpf-test/pullrequest/2485?_a=overview

**Comment by rladuca** [comment]
Re-enabling the splash screen tests will require bumping the version of Core the test repo builds on.  This can sometimes cause issues with tests, I'll iterate on that once the Preview validations are done so the team isn't overloaded.

**Comment by rladuca** [comment]
JumpList problems are well known issues since WinBlue.

I'm going to split them into a separate issue and add the comments from internal bugs related to it (https://devdiv.visualstudio.com/DevDiv/_workitems/edit/126798).

**Comment by rladuca** [comment]
I tried bumping the test repo to build off the latest Core dailies.  This can't be done right now, as we are blocked by #1552 .  We can look at this again once that is fixed.
